server/tests/test_responses_api.py

```
@@ -0,0 +1,103 @@
+            data = response.json()
+            print(f"Response: {json.dumps(data, indent=2)}")
+            content = data["choices"][0]["message"]["content"]
+            parsed = json.loads(content)
+            print(f"Parsed JSON: {parsed}")
+            assert parsed["capital"].lower() == "berlin"
+            assert parsed["country"].lower() == "germany"
+            print("JSON schema test PASSED")
+        else:
+            print(f"JSON schema test FAILED: {response.text}")
+
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)